models/stg_shopify__order.sql

```diff
@@ -36,7 +36,6 @@ final as (
         total_line_items_price_set,
         total_price,
         total_price_set,
-        total_price_usd,
```

Comment:
Just make sure to remove these from the corresponding get column macros as well!

Reply:
And `processing method` too

Reply:
Thanks! I missed removing these fields. Just updated.